### PR TITLE
Remove security warning (SECURITY-835) from plugin

### PR DESCRIPTION
**Pull request by nikhil-dabhade**
The issue SECURITY-835 was resolved in the plugin repo on Aug 6th, 2019 with this code commit for version 1.26 of the plugin:
2d8cfe3cdd299c152ca5d13aada51cb69d0fec88

The plugin has been released multiple times since then.

Link to plugin repo and the commit that resolved the issue:
https://github.com/jenkinsci/aws-device-farm-plugin/commit/2d8cfe3cdd299c152ca5d13aada51cb69d0fec88

### CODE REVIEW

File: resources/warnings.json [hunk 1]
```diff
@@ -4151,7 +4151,8 @@
     "url": "https://jenkins.io/security/advisory/2019-04-03/#SECURITY-835",
     "versions": [
       {
-        "pattern": ".*"
+        "lastVersion": "1.25",
+        "pattern": "(1[.][0-2][0-5])"
```
Comment (daniel-beck) [inline resources/warnings.json:4155]: This pattern would not consider 1.3 through 1.9, or 1.16 through 1.19 to be vulnerable.
```suggestion
        "pattern": "(1[.]1[0-9]|1[.]2[0-5]|1[.][3-9])(|[.-].+)"
```

**Review by daniel-beck — CHANGES_REQUESTED** [review]
Thanks for letting us know! We don't generally monitor new releases of plugins for which we announce unresolved issues, so we missed this.

We'll review the fix and then merge this PR once amended if we confirm the fix is complete.

**Comment by daniel-beck** [comment]
I confirm the issue is resolved (although the inconsistent treatment of `akid` not being masked on the UI, but stored encrypted, seems odd); just the regex needs fixing.

File: resources/warnings.json (hunk 1 above)
Comment (nikhil-dabhade) [reply resources/warnings.json:4155]: Thank you very much @daniel-beck for the suggestion and apologies as this slipped through my notifications.
This looks good to me. Accepting the change and merging.

**Comment by nikhil-dabhade** [comment]
Thank you @daniel-beck for the suggestions and sorry for the extreme delay as this one slipped my eye in notifications.
I accepted your changes and updated the pull request.

**Comment by nikhil-dabhade** [comment]
@daniel-beck Anything more that I help with to get this going?

**Comment by daniel-beck** [comment]
@nikhil-dabhade Thanks for the reminder! Looks like I missed your previous comment, sorry about that.

**Review by daniel-beck — APPROVED** [review]

**Pull request merged**